tgmc.dme

```diff
@@ -1,1369 +1,1368 @@
-// DM Environment file for baystation12.dme.

```

Comment:
This is not right, you must have changed the lineendings or something. You will have to fix this, or it will break git blame